swift/ios-shoppe-demo/Resources/Constants.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  Constants.swift
+//  ios-shoppe-demo
+//
+//  Created on 8/25/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
+
+import Foundation
+
+typealias nib = K.Nib
+typealias cellID = K.CellID
+
+/// Hard coded constants
+struct K {
+    struct Title {
+        static let orderCart = "Order Cart"
+        static let shoppe = "iOS Shoppe"
+    }
```

Comment:
just curious, why are these structs not just enums?

Reply:
I thought about that too lol it would be even easier. No particular reason.